FoursquareTop/Constants/Images.swift

```diff
@@ -8,6 +8,8 @@
   typealias Image = NSImage
 #endif
 
+// swiftlint:disable file_length
```

Comment:
hmm... I haven't add it by hand that for sure... let me remove it and check if it appears again.